cc/tflite_micro/generated/third_party/kissfft/COPYING

```diff
@@ -0,0 +1,11 @@
+Copyright (c) 2003-2010 Mark Borgerding
```

Comment:
It's placed in tflm for future open sourcing that supports audio frontend library.
(The audio frontend library is a collection of TF, TFLite and TFLM operators used to compose frontends for audio models.)

For the time being, we don't use it so I don't put kissfft headers in the BUILD rule.
As aforementioned, I don't want to touch any bit of generated source code, otherwise we are increasing complexity of upgrading/downgrading tflm as well as maintenance cost.

Reply:
As I've moved generated sources to third_party/ via tool script, kissfft is ignored for the time being given no one is using it.